src/Microsoft.SqlTools.Migration/Contracts/GetArmTemplateRequest.cs

```diff
@@ -3,14 +3,15 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
+using System.Collections.Generic;
 using Microsoft.SqlTools.Hosting.Protocol.Contracts;
 
 namespace Microsoft.SqlTools.Migration.Contracts
 {
     public class GetArmTemplateRequest
     {
         public static readonly
-            RequestType<string, string> Type =
-                RequestType<string, string>.Create("migration/getarmtemplate");
+            RequestType<string, List<string>> Type =
+                RequestType<string, List<string>>.Create("migration/getarmtemplate");
```

Comment:
From SQL NuGet output is List<SqlArmTemplate>, are we doing any handling to get in List <String>? 
or automatically it is getting handled?

Reply:
this contract is between sqltoolsservice and ADS.